packages/vuetify/src/components/VCalendar/util/props.ts

```diff
@@ -109,10 +113,33 @@ export default {
       default: 'month',
     },
     value: {
-      type: String,
+      type: [String, Number, Date] as PropType<string | number | Date>,
       validate: validateTimestamp,
     },
   },
+  category: {
+    categories: {
+      type: [Array, String],
+      default: '',
+    },
+    categoryHideDynamic: {
+      type: Boolean,
+      default: false,
+    },
+    categoryShowAll: {
+      type: Boolean,
+      default: false,
+    },
+    categoryForInvalid: {
```

Comment:
Strings are empty by default, `''`

Reply:
I did a simple test and they appear to be undefined